src/fontra/views/editor/editor.js

```diff
@@ -2248,6 +2248,7 @@ export class EditorController {
   }
 
   async doAddGuideline(global = false) {
+    this.visualizationLayersSettings.model["fontra.guidelines"] = true;
```

Comment:
I feel that perhaps this should be done after the dialog. Not 100% sure. Did you consider that?

Reply:
Ah, you mean only get active when you really add a guideline and not if you cancel it?! I see your point. 
I tried it and I found it useful to have it displayed immediately, because in the case you simply forgot to active 'show guidelines' and you want to add a new guideline at the same position where you have already added a guideline but you forgot about it, you will then see it in the background, which will be a reminder: 'Ah. Oh. Yes, there is already a guideline. I simply missed to show the guidelines'. In that case you can cancel adding guidelines and see/adjust the existing guidelines. 
But if you want to have it different I am happy to change it.

Reply:
That’s a good argument, let’s keep it this way!